scanpointgenerator/generators/powertermgenerator.py

```diff
@@ -0,0 +1,84 @@
+from scanpointgenerator.core import Generator
+from scanpointgenerator.compat import np
+
+def to_list(value):
```

Comment:
You don't seem to use this, does this mean that you only ever need a power term for a single axis, or could you need one in multiple axes like we have for LineGenerator?

Reply:
What would that look like? Would x0 and y0 be n-dimensional and x be n*length dimensional?

Reply:
Thanks for pointing that out, I forgot to delete as I am only considering a single axis.
Having said that, I had another look at LineGenerator, and I think it is easy enough to extend this to multiple axes. Not sure how useful that would be, but I guess there's no good reason to limit functionality at this level. Will investigate further tomorrow.

Reply:
@thomascobb n-dimensions now supported